pyanaconda/ui/tui/spokes/storage.py

```diff
@@ -361,9 +361,13 @@ def run_dasdfmt(self, to_format=None):
                 continue
 
         # need to make devicetree aware of disk changes
-        threadMgr.add(AnacondaThread(name=THREAD_STORAGE, target=storageInitialize,
-                                     args=(self.storage, self.data, self.storage.devicetree.protectedDevNames)))
-        self._initialize()
+        self.storage.devicetree.populate()
+        if not flags.automatedInstall:
+            # reinit storage
+            threadMgr.add(AnacondaThread(name=THREAD_STORAGE, target=storageInitialize,
+                                         args=(self.storage, self.data, self.storage.devicetree.protectedDevNames)))
+            # update the summary screen with the changes
+            self._initialize()
```

Comment:
It's necessary to wait on this thread in manual installations because otherwise the screen won't refresh with the updated config. (Obviously, it's not necessary in ks, since you don't see the storage spoke.)